firestore-send-email/functions/src/index.ts

```diff
@@ -340,7 +363,7 @@ async function processWrite(change) {
 
 export const processQueue = functions.handler.firestore.document.onWrite(
   async (change) => {
-    initialize();
+    await initialize();
```

Comment:
did we have a race condition here previously?

Reply:
Not a race condition, but it needs to be async for the integration test to work [here](https://github.com/firebase/extensions/pull/527/files/7f769bd1e0d33dcd18ddf122a609d98f9bfef9b8#diff-f36d0ed1929c306fadc22ba963e485ebcc96248da84b6b0d57cc1df7c0b37f3bR48-R67)